vyper/builtins/functions.py

```diff
@@ -1479,8 +1480,8 @@ def evaluate(self, node):
         value, shift = [i.value for i in node.args]
         if value < 0 or value >= 2 ** 256:
             raise InvalidLiteral("Value out of range for uint256", node.args[0])
-        if shift < -(2 ** 127) or shift >= 2 ** 127:
-            raise InvalidLiteral("Value out of range for int128", node.args[1])
+        if shift < -255 or shift > 255:
```

Comment:
i think a shift amount of 256 might be acceptable here, in any case if the literal is too large later it will get caught too